chain/params.go

```diff
@@ -1,18 +1,54 @@
 package chain
 
 import (
+	"errors"
 	"math/big"
+	"sort"
+	"strconv"
 
 	"github.com/0xPolygon/polygon-edge/types"
 )
 
+var (
+	// ErrBurntContractAddressMissing is the error when a contract address is not provided
+	ErrBurntContractAddressMissing = errors.New("burnt contract address missing")
+)
+
 // Params are all the set of params for the chain
 type Params struct {
 	Forks          *Forks                 `json:"forks"`
 	ChainID        int                    `json:"chainID"`
 	Engine         map[string]interface{} `json:"engine"`
 	Whitelists     *Whitelists            `json:"whitelists,omitempty"`
 	BlockGasTarget uint64                 `json:"blockGasTarget"`
+
+	// Governance contract where the token will be sent to and burnt in london fork
+	BurntContract map[string]string `json:"burntContract"`
```

Comment:
As @vcastellm mentioned in his review, should this be renamed to `BurnContract`, instead of `BurntContract`? Here and all the other places across the codebase (refer to it as burn, instead of burnt, because it is more semantically correct). Agree?

Reply:
Right, fixed